.github/workflows/pr_basic_compilation_check.yml

```diff
@@ -32,8 +32,13 @@ jobs:
       - name: Compile "enclave-tls"
```

Comment:
Please use `if: ${{ !contains(matrix.sgx, 'alinux2') }}`  to compile enclave-tls  on`ubuntu and centos` system.